package.json

```diff
@@ -58,7 +677,7 @@
         "controlType": "textbox"
```

Comment:
This is not user friendly. It should be a search like the international weather applet